Doc update ps cmd (#11243)
nrfconnect/sdk-nrf: doc/nrf/working_with_nrf/nrf70/developing/powersave.rst

  nRF70 Series devices operate in this mode by default.
  The user or application can change the device to the Permanent Active mode using the ``NET_REQUEST_WIFI_PS`` network management API.

  Further the Wi-Fi® interface can be configured in one of the following dynamic Power Save modes

>         * **Normal PS:** Delivery Traffic Indication Message

Reviewer: ```suggestion
        * **Normal Power Save:** Delivery Traffic Indication Message (DTIM).
```

          DTIM is the default configuration and enabled by the application based on the traffic profile.

>         * **Extened PS:** Listen interval based powwe save

Reviewer: ```suggestion
        * **Extended Power Save:** Listen interval-based power save mode.
```

>           The Listen interval based Power Save mode is supported to enable devices to sleep longer than the DTIM period.

Reviewer: ```suggestion
          The Listen interval based-Power Save mode is supported to enable devices to sleep longer than the DTIM period.
```

>           It is typically low, so, not useful for power saving.

Reviewer: ```suggestion
          It is typically low and not useful for power saving.
```

>         * **Deep sleep:** Target Wake Time (TWT) power save

Reviewer: ```suggestion
        * **Deep sleep:** Target Wake Time (TWT) power save.
```

  nRF70 Series devices operate in this mode by default.
  The user or application can change the device to the Permanent Active mode using the ``NET_REQUEST_WIFI_PS`` network management API.

Further the Wi-Fi® interface can be configured in one of the following dynamic Power Save modes:

* **Normal Power Save:** Delivery Traffic Indication Message (DTIM).
  DTIM is the default configuration and enabled by the application based on the traffic profile.
* **Extended Power Save:** Listen interval-based power save mode.
  The Listen interval based-Power Save mode is supported to enable devices to sleep longer than the DTIM period.
  It is typically low and not useful for power saving.
* **Deep sleep:** Target Wake Time (TWT) power save.

>   TWT can be enabled if the connected access point is Wi-Fi 6 capable.

Reviewer 2: Formatting is bit off here, Can you please check?

Reviewer: Yes, it could be @ajayparida that the second sentence is has too much indentation? And needs to be like the green marked area in this screenshot/same formatting the as the list that starts on line 60.
https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-11243/nrf/_sources/working_with_nrf/nrf70/developing/powersave.rst.txt
![image](https://github.com/nrfconnect/sdk-nrf/assets/51916561/a84a720c-8b58-4cba-bdcf-a5915f207e2c)

   WMM Power Save mode

Legacy Power Save mode is the default option in nRF70 Series devices.
The user or application can configure the WMM mode using the ``NET_REQUEST_WIFI_PS`` network management API.
The WMM mode does not deliver a significant performance or power difference when compared to Legacy mode in Dynamic Power Save operation.

The average power consumption of the device is affected by the DTIM period.
The typical value is three beacons, for example, 307 ms for a beacon period of 100-time units.
The higher DTIM period results in increased power saving and higher latency to the application data.

.. _ug_nrf70_developing_powersave_extended_ps:

Extended Power Save
*******************

> Listen Interval based power save mode is supported to enable devices to sleep longer than DTIM period in absence of TWT session. Device wake up interval is adjusted

Reviewer: ```suggestion
The Listen Interval based Power Save mode is supported to enable devices to sleep longer than DTIM period in absence of TWT session. The device wake up interval is adjusted
```

   WMM Power Save mode

Legacy Power Save mode is the default option in nRF70 Series devices.
The user or application can configure the WMM mode using the ``NET_REQUEST_WIFI_PS`` network management API.
The WMM mode does not deliver a significant performance or power difference when compared to Legacy mode in Dynamic Power Save operation.

The average power consumption of the device is affected by the DTIM period.
The typical value is three beacons, for example, 307 ms for a beacon period of 100-time units.
The higher DTIM period results in increased power saving and higher latency to the application data.

.. _ug_nrf70_developing_powersave_extended_ps:

Extended Power Save
*******************

> The Listen Interval based Power Save mode is supported to enable devices to sleep longer than DTIM period in absence of TWT session.

Reviewer: Sorry for any previous confusion. Should be "Listen interval" when referring to the name of the setting.

> The Listen Interval based Power Save mode is supported to enable devices to sleep longer than DTIM period in absence of TWT session.

Reviewer 2: "This feature" instead of "This features"?

The device wake up interval is adjusted to nearest multiple of DTIM Period.
For example, listen interval of 10, device wakes up for every 9th beacon if DTIM is 3 and 8th beacon if DTIM is 4.

> Device wakes up for configured Listen interval instead of DTIM beacon and will miss chance of receiving broadcast and multicast frames which get scheduled after DTIM

Reviewer: ```suggestion
The device wakes up for the configured Listen interval instead of the DTIM beacon.
It will miss the chance of receiving a broadcast and the multicast frames which are scheduled after the DTIM beacon.
```

> beacon.

Reviewer: Remove, added to new sentence in previous suggestion.

> Listen interval-based power needs to be enabled only if application can handle loss broadcast frames.

Reviewer: ```suggestion
The Listen interval-based power needs to be enabled only if the application can handle loss broadcast frames.
```

> Chances of losing broadcast frames increases with higher listen interval.

Reviewer: ```suggestion
Chances of losing broadcast frames increases with higher listen intervals.
```

> Following parameters control functionality of extended power save operation.

Reviewer: ```suggestion
The following parameters control the functionality of the extended Power Save operation.
```

Reviewer 2: I think "power save" operations instead of capitalizing **Power Save** as we are talking about operations not mode here?

Listen interval
===============

> Listen interval is a field present in association request frame and it indicates how frequently the device will wake to listen TIM.

Reviewer: ```suggestion
The Listen interval is a field that is present in an association request frame.
It indicates how frequently the device will wake to listen to the TIM.
```

Reviewer: Should the "TMI" here be "DTMI"?

> Larger value enable the devices to save more power by sleeping longer periods.

Reviewer: ```suggestion
A larger value enables the devices to save more power by sleeping for longer periods.
```

> Larger listen interval adds burden on AP to buffer frames much longer to device and this may force AP to decline Connection.

Reviewer 3: ```suggestion
Larger listen interval adds burden on AP to buffer frames much longer to device and this may force AP to decline Connection and also if the buffered frames exceed MPDU/MSDU lifetime then they will be silently dropped by the AP causing frame loss.
```

> Typical listen interval is 10 beacons.

Reviewer: ```suggestion
The typical listen interval is 10 beacons.
```

> Listen interval can be set using ``NET_REQUEST_WIFI_PS`` API and should be called before connection to AP.

Reviewer: ```suggestion
The Listen interval can be set by using the ``NET_REQUEST_WIFI_PS`` API, and should be called before connecting to the AP.
```

Wakeup mode
===========

> Device can be configured to wakeup for DTIM or Listen interval using ``NET_REQUEST_WIFI_PS`` API.

Reviewer: ```suggestion
The device can be configured to wake up for the DTIM or Listen interval by using the ``NET_REQUEST_WIFI_PS`` API.
```

Reviewer: Is "wake up" used as a verb here? If so, then please use the suggestion. If it is meant to be a noun, when keep as one word.

> Device can switch from DTIM to Listen interval based power save at run time seamlessly.

Reviewer: ```suggestion
The device can switch from the DTIM to the Listen interval-based Power Save at runtime seamlessly.
```

The following figure illustrates change of wakeup mode from DTIM to Listen interval.

.. figure:: images/nRF70_ug_change_wakeup_mode_from_dtim_to_li.png

>    :alt: Change Power Save wakeup mode from DTIM to Listen interval

Reviewer: ```suggestion
   :alt: Change Power Save wakeup mode from DTIM to Listen interval
```

   Change Power Save wakeup mode (DTIM to Listen interval)

> The following figure illustrates change of wakeup mode from Listen interval to DTIM.

Reviewer: ```suggestion
The following figure illustrates the change in wakeup mode from the Listen interval to the DTIM.
```

.. figure:: images/nRF70_ug_change_wakeup_mode_from_li_to_dtim.png
   :alt: Change Power Save wakeup mode from Listen interval to DTIM

>    Change Power Save wakeup mode (Listen interval to DTIM)

Reviewer: ```suggestion
   Change Power Save wakeup mode from Listen interval to DTIM
```
